fabfile.py

```diff
@@ -38,12 +36,15 @@ def write_nginx_config(config0):
 def bootstrap(ctx, host):
   c = Connection(host=host, user=USER)
   c.config = ctx.config
+  email = c.config.get('email')
   c.run('mkdir --parents /opt/mes-aides')
-  c.run('apt-get install --assume-yes htop openssh-client python3-pip rsync vim')
-  c.local('rsync -r . %s@%s:/opt/mes-aides/ops --exclude .git --exclude .venv37 --exclude .vagrant -v' % (USER, host))
+  c.run('apt-get update')
+  c.run('apt-get install --assume-yes htop openssh-client libffi-dev rsync vim')
+  python(c)
+  c.local('rsync -r . %s@%s:/opt/mes-aides/ops %s -v' % (USER, host, RSYNC_EXCLUDE))
```

Comment:
Je sais que c'était déjà là avant, mais je pense qu'en terme de lisibilité ça serait beaucoup plus clair de passer aux f-strings plutôt que de rester sur les %-formatting.

Ex :
```suggestion
  c.local(f"rsync -r . {USER}@{host}:/opt/mes-aides/ops {RSYNC_EXCLUDE} -v")
```